en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -277,13 +277,15 @@ Run any of the following CTL commands to get keys for the API.
             
         -   Required :  
             `--environment` or `-e` : Key generation environment  
-            `--name` or `-n` : API to enerate keys for  
-            `--version` or `-v` : Version of the API  
-            `--provider` or `-r` : Provider of the API   
+            `--name` or `-n` : API or API Product to enerate keys for   
```

Comment:
Shall we change this to "generate"?

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3097/commits/74cdaa1323d12a99abd038f41a690914b8d50ab5

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/3098/commits/edd8ae4c020155b3c9d8e4465a923a031e8f9472 for 3.2.0